lib/Software/Security/Policy.pm

```diff
@@ -131,7 +142,11 @@ sub git_url { (defined $_[0]->{git_url} ? $_[0]->{git_url} :
                 'SECURITY.md')) }
 
 
-sub support_years { $_[0]->{support_years} || '10'}
+sub perl_support_years { (defined $_[0]->{perl_support_years} ? $_[0]->{perl_support_years} :
+            (defined $_[0]->{support_years} ? $_[0]->{support_years} : '10'))}
+
+sub support_years { (defined $_[0]->{support_years} ? $_[0]->{support_years} :
+            (defined $_[0]->{perl_support_years} ? $_[0]->{perl_support_yearl} : '10'))}
```

Comment:
Allowing both of these attributes and allowing them to be different doesn't seem ideal.

I think I'd either normalize them in the constructor (which could die if both were provided) or make the accessors equivalent.

Alternatively, this is early enough in the module's life that you could just break compat and drop `support_years`.

Reply:
I was thinking about that and you are likely right.  I will revise.